test/hotspot/jtreg/TEST.groups

```diff
@@ -54,7 +54,8 @@ hotspot_serviceability = \
 
 hotspot_valhalla = \
   runtime/valhalla \
-  compiler/valhalla
+  compiler/valhalla \
+  serviceability/jvmti/Valhalla
```

Comment:
For consistency, should the we rename the test sub-folder from `Valhalla` to `valhalla` ?
This comment is not about this current fix though.